scripts/generate_go.sh

```diff
@@ -7,8 +7,10 @@ export PATH="$PATH:$(go env GOPATH)/bin"
 # make the generated folder if it doesnt already exist
 mkdir -p generated/go
 
+proto_files=$(find messages -type f -name "*.proto")
+
 protoc \
 --proto_path=messages \
 --go_out=generated/go \
 --go_opt=paths=source_relative \
-metrics/node_stats.proto common.proto pump/pump_stats.proto pump/mixing_stats.proto
```

Comment:
@SnackkOverflowError when u get some time could u make the script glob for proto files so that we don't need to re-add them here? i am gonna revert this change in this pr cause I don't have a way to test it oop